tests/unit/objects/test_ensure_type_hints.py

```diff
@@ -0,0 +1,84 @@
+"""
+Ensure type-hints are setup correctly and detect if missing functions.
+
+Original notes by John L. Villalovos
+
+"""
+import inspect
+from typing import Tuple, Type
+
+import toml
+
+import gitlab.mixins
+import gitlab.v4.objects
+
+
+def pytest_generate_tests(metafunc):
+    """Find all of the classes in gitlab.v4.objects and pass them to our test
+    function"""
+
+    # Ignore any modules that we are ignoring in our pyproject.toml
+    excluded_modules = set()
+    with open("pyproject.toml", "r") as in_file:
+        pyproject = toml.load(in_file)
+    overrides = pyproject.get("tool", {}).get("mypy", {}).get("overrides", [])
+    for override in overrides:
+        if override.get("ignore_errors") is True:
+            for module in override.get("module", []):
+                if module.startswith("gitlab.v4.objects"):
+                    excluded_modules.add(module)
```

Comment:
Would this work also?
```suggestion
    for override in overrides:
        if not override.get("ignore_errors"):
            continue
        for module in override.get("module", []):
            if module.startswith("gitlab.v4.objects"):
                excluded_modules.add(module)
```

Reply:
Done!